zds/tutorialv2/views/display.py

```diff
@@ -84,6 +85,9 @@ def get_context_data(self, **kwargs):
             queryset_pagination = PublishedContent.objects.filter(
                 content_type=self.current_content_type, must_redirect=False
             )
+            
+        if self.current_content_type == 'OPINION':
+            queryset_pagination = queryset_pagination.filter(content__sha_picked=F('sha_public'))
```

Comment:
Actuellement tu proposes ce code là (les contenus précédents et suivants ne sont calculés que pour les billets) : 

```
Si billet ou article : 
    queryset = ...
Si billet : 
    queryset = ...
    contenu d'avant = ...
    contenu d'après = ...
```

Or il faudrait ceci (les contenus précédents et suivants sont calculés pour les billets et les articles ) : 

```
Si billet ou article : 
    queryset = ...
    Si billet : 
        queryset = ...
    contenu d'avant = ...
    contenu d'après = ...
```

Il faut donc décaler d'une indentation ton bloc !

```suggestion
            if self.current_content_type == 'OPINION':
                queryset_pagination = queryset_pagination.filter(content__sha_picked=F('sha_public'))
```

Reply:
oh je vois